app/views/registers/show.html.haml

```diff
@@ -86,7 +86,9 @@
                     %thead
                       %tr{data: {"click-events" => true, "click-category" => "Register Table", "click-action" => "Sort"}}
                         - @register.register_fields.each do |field|
-                          %th{class: "#{field['field']}"}= field['cardinality'] == '1' ? records_table_sort_link(field['field'], request.query_parameters) : field['field']
+                          %th{class: "#{field['field']}"}
+                            = field['cardinality'] == '1' ? records_table_sort_link(field['field'], request.query_parameters) : field['field']
+                            = link_to '?', register_field_url(@register.slug, field['field']), class: 'info-icon', remote: true, "aria-label" => "What does #{field['field']} mean?"
```

Comment:
Not sure of the merits of `aria-label` vs hidden off-screen text?

Reply:
Replaced with a visually hidden span element 👍